blueprint/index.md

```diff
@@ -240,156 +241,151 @@ The Microsoft Teams video session URL is sent as an SMS to the customer from Gen
-
-  ![Expand the First Data Action](images/mapFirstDataAction.png "Expand the First Data Action")
-
-8. Expand the input variables for the First Data Action
+7. From the Category list, select the Teams meeting web services data action. From the Data Action list, select the Teams meeting data action.
 
-  ![Expand Input Variables](images/mapFirstDataActionExpandInputVariables.png "Expand Input Variables")
+   ![Expand the First Data Action](images/mapFirstDataAction.png "Expand the First Data Action")
 
-9. Type the desired value in the **user** input variable.
-
-  :::primary
-  **Note:** The variable value in the example below will create a Teams meeting through the agent's Teams account.  For this to work, the agent's Genesys Cloud email address must match their Teams email address.  If you'd like to use the same Teams account to create the Teams meeting regardless of which agent is on the interaction, you can define a static value here.  It can be either the email address or object ID of a person in the Azure Activity Directory where your app is registered.
-  :::
+8. Expand the input variables for the first data action and enter the user input variable. The variable can be either an agent's Teams account or the Teams account that must be used regardless of which agent is on the interaction.
+   :::primary
+   **Note:** When you enter an agent's Teams account, the agent's Genesys Cloud email address must match their Teams email address. When you define a general Teams account, it can be either an email address or an object ID of an agent in the Azure Activity Directory.
+   ::: 
 
-  ![Map First Data Action Input Variable](images/mapFirstDataActionDefineUserInputVariable.png "Map First Data Action Input Variable")
+      ![Map First Data Action Input Variable](images/mapFirstDataActionDefineUserInputVariable.png "Map First Data Action Input Variable")
 
-10. Define static values for the remaining input variables.
+9. Define the static values for the input variables, `startTime`, `endTime`, and `timeZone`. 
 
-  :::primary
-  **Note:** Using the values in this example will create ISO-8601 formatted timestamps.  The startTime and endTime parameters can be the same and it can occur in the past.  If you would like to define your own static timestamps for these variables, you are welcome to do so.
-  :::
+   :::primary
+   **Note:** You can enter the same value for `startTime` and `endTime` parameters and the timestamp can be in the past. Also, define your own timestamp format. The example shows the value to create the ISO-8601 format timestamp. 
+   :::
 
-  ![Define Static Input Variables](images/mapFirstDataActionDefineInputVariables.png "Define Static Input Variables")
+   ![Define Static Input Variables](images/mapFirstDataActionDefineInputVariables.png "Define Static Input Variables")
 
-11. Expand the second data action.
+10. Expand the second data action that sends the SMS.
```

Comment:
Does the second data action have a predefined name? Or is it user-defined? Also, "...SMS message."